packages/fetchai/skills/aries_faber/handlers.py

```diff
@@ -47,6 +48,8 @@
 ADMIN_COMMAND_CREATE_INVITATION = "/connections/create-invitation"
 ADMIN_COMMAND_STATUS = "/status"
 HTTP_COUNTERPARTY = "HTTP Server"
+FABER_ACA_IDENTITY = "Faber_ACA"
+LEDGER_URL = "http://127.0.0.1:9000"
```

Comment:
These should all be "default" constants, and then be doubled by values settable via the yaml config file. So you can configure it with alternative values (see our other demos, in particular the strategy.py files for examples of what I mean)

Reply:
Done